flink-cdc-common/src/main/java/org/apache/flink/cdc/common/schema/Schema.java

```diff
@@ -25,6 +25,8 @@
 import org.apache.flink.cdc.common.types.RowType;
 import org.apache.flink.cdc.common.utils.Preconditions;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
We may use `org.apache.flink.cdc.common.utils.StringUtils#isNullOrWhitespaceOnly`